docs/hints-for-use-with-docker.md

```diff
@@ -0,0 +1,52 @@
+# Hints for using ORT with Docker
+
+- [Hints for using ORT with Docker](#hints-for-using-ort-with-docker)
+  - [General Hints](#general-hints)
+    - [Mount current working directory or directory relative to working directory](#mount-current-working-directory-or-directory-relative-to-working-directory)
+  - [Common Issues](#common-issues)
+    - [Docker build fails with a "SSL Handshake" error](#docker-build-fails-with-a-ssl-handshake-error)
+    - [Authenticating with a private git repository fails](#authenticating-with-a-private-git-repository-fails)
+
+## General Hints
+
+### Mount current working directory or directory relative to working directory
+
+Use `$(pwd)` on unix-based systems, `${PWD}` for PowerShell on Windows:
```

Comment:
You are right, better use the environment variable for both, I wasn't aware it existed on Unix as well. The curly braces are there, because PS interprets the `:` required for docker as part of the variable name, but I believe in this instance it can also be worked around by appending "/" which *should* work both in Unix Shells and PowerShell. I'll verify and update.